app/models/user.js

```diff
@@ -0,0 +1,26 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      firstName: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
```

Comment:
It's a good practice to add all fields in the model, so we don't have to look at the migration once it is run. Add the field `id`.